components/script/script_runtime.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(dead_code)]
```

Comment:
I said:
* `#![allow(dead_code)` (this has a different meaning than `#[allow(dead_code)]`
* place it after the license header comment

This doesn't build right now.

Reply:
I am sorry, I'd misread the comments. I have fixed that lint and updated the PR.